server/config/releases.exs

```diff
@@ -1,28 +1,35 @@
 import Config
 
-app_port = System.get_env("PORT") || 4000
 app_hostname = System.get_env("HOSTNAME") || "localhost"
-db_user = System.get_env("DB_USER") || "postgres"
-db_password = System.get_env("DB_PASSWORD") || "postgres"
+app_port = System.get_env("PORT") || "4000"
```

Comment:
Should this one be a number?

Reply:
Seems to crash with a number:
```
ERROR! Config provider Config.Reader failed with:
** (ArgumentError) argument error
    :erlang.binary_to_integer(4000)
    (stdlib 3.13) erl_eval.erl:680: :erl_eval.do_apply/6
    (stdlib 3.13) erl_eval.erl:888: :erl_eval.expr_list/6
    (stdlib 3.13) erl_eval.erl:240: :erl_eval.expr/5
    (stdlib 3.13) erl_eval.erl:232: :erl_eval.expr/5
    (stdlib 3.13) erl_eval.erl:233: :erl_eval.expr/5
    (stdlib 3.13) erl_eval.erl:888: :erl_eval.expr_list/6
    (stdlib 3.13) erl_eval.erl:240: :erl_eval.expr/5

{"init terminating in do_boot",{badarg,[{erlang,binary_to_integer,[4000],[]},{erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,680}]},{erl_eval,expr_list,6,[{file,"erl_eval.erl"},{line,888}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,240}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,232}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,233}]},{erl_eval,expr_list,6,[{file,"erl_eval.erl"},{line,888}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,240}]}]}}
init terminating in do_boot ({badarg,[{erlang,binary_to_integer,[4000],[]},{erl_eval,do_apply,6,[{_},{_}]},{erl_eval,expr_list,6,[{_},{_}]},{erl_eval,expr,5,[{_},{_}]},{erl_eval,expr,5,[{_},{_}]},{erl

Crash dump is being written to: erl_crash.dump...done
```

Works fine with a string though (the build fail is something different, looking into it)

Reply:
Cool 👍